crates/client/src/client.rs

```diff
@@ -938,8 +938,10 @@ impl FuelClient {
     pub async fn submit_and_await_status<'a>(
```

Comment:
I think we can leave the signature of this function and just accept new field in the `submit_and_await_status_opt`=)

Reply:
Done